coopterrones/color-palette
# Feature/randomizer button

#### What's this PR do?
- creates randomizer button component 
- creates randomizer button functionality 
- layout for randomize button styling

#### Where should the reviewer start?
- line 39 `App.js` creates the functionality calling getColors
- line 48 `App.js` adds the component into the main page
- line 15 `ColorCard.js` uses hexCode as color for window on card now -- rgbVal was not updating
- line 1 `RandomizerButton.scss` creates stylign for RandomizerButton 
- line 1 `RandomizerButton.js` creates the RandomizerButton component 

#### How should this be manually tested?
- check that the randomizer button appears on load
- onClick should generate a new set of colors with updated labeling
